File: week-02/day-01/draw_diamond.py
# Write a program that reads a number from the standard input, then draws a
# diamond like this:
#
#
#    *
#   ***
#  *****
# *******
#  *****
#   ***
#    *
#
# The diamond should have as many lines as the number was
a = int(input())
b = a - 1
c = a * 2 -1
for i in range(1, a + a * 1, 2):
    print(' ' * b + '*' * i )
    b -= 1
for o in range(1, a + a * 1, 2):
    print(' ' * (o-2) + '*' * c )
    c -= 2

File: week-07/day-01/draw-diamond.js
'use strict';

var lineCount = 7;



// Write a program that draws a
// diamond like this:
//
//
//    *
//   ***
//  *****
// *******
//  *****
//   ***
//    *
//
// The diamond should have as many lines as lineCount is

let star = '*';

for (let i = Math.floor(lineCount / 2); i > 0; i--) {
  console.log(' '.repeat(i) + star);
  star += '**';
}

for (let i = 0; i <= Math.floor(lineCount / 2); i++) {
  console.log(' '.repeat(i) + star.substr(i * 2 , star.length));
}
